src/Solvers/Random.jl

```diff
@@ -1,8 +1,54 @@
+using OptimizedEinsum: removedsize, ssa_path_cost, ContractionPath
+using OptimizedEinsum.Solvers: greedy_choose_simple!, greedy_choose_thermal!
+using Base: @kwdef
+using Distributions: Normal, truncated
+using Random: seed!
+
 abstract type RandomSolver <: Solver end
 
-struct RandomGreedy <: RandomSolver
-    temperature::Float32
-    rel_temperature::Bool
-    nbranch::Int
-    cost_fn::Core.Function
+@kwdef struct RandomGreedy <: RandomSolver
+    temperature::Float32 = 1.
+    rel_temperature::Bool = true
+    nbranch::Int = 8
+    repeats::Int = 128
+    choose_fn::Function = (x, y) -> greedy_choose_thermal!(x, y, nbranch, temperature, rel_temperature)
+    cost_fn::Function = (a, b, size, keep) -> rand(truncated(Normal(1.0, 0.01),0,10),1)[1]*removedsize(a, b, size, keep)
```

Comment:
This smells bad. Just pass `removedsize` and add the noise in the code.
```suggestion
    cost_fn::Function = removedsize
```